openedxscorm/scormxblock.py

```diff
@@ -277,7 +278,8 @@ def extract_package(self, package_file):
                             os.path.relpath(zipinfo.filename, root_path),
                         )
                         self.storage.save(
-                            dest_path, scorm_zipfile.open(zipinfo.filename),
+                            dest_path,
+                            ContentFile(scorm_zipfile.open(zipinfo.filename).read()),
```

Comment:
I believe this can be shortened to `ContentFile(scorm_zipfile.read(zipinfo.filename))`, right?

Reply:
right :+1: